Dockerfile

```diff
@@ -1,11 +1,11 @@
 ARG NGINX_VERSION=1.16.1
-ARG NGINX_RTMP_VERSION=1.2.1
+ARG NGINX_RTMP_VERSION=2.0.0.6
 ARG FFMPEG_VERSION=4.2.1
 
 
 ##############################
 # Build the NGINX-build image.
-FROM alpine:3.8 as build-nginx
+FROM alpine:latest as build-nginx
```

Comment:
Can we pin this to `1.11` instead of `latest`?

Reply:
done